src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -67,10 +100,19 @@ public HashSet<int> Start(ApplicationData applicationData)
             return selectedIndexes;
         }
 
+        private void Accept(){
+            Application.RequestStop();
+        }
+
+        private void Close(){
+            _cancelled = true;
+            Application.RequestStop();
+        }
```

Comment:
These could be refactored to be in `ConsoleGuiWindow` since Application is static. Then you'd just need to make `_cancelled` set-able.

Reply:
Thought about that, but `ConsoleGuiWindow` will go away once we have `StatusBar` because `StatusBar` provides `keyEvent` handling.